tests/test_client.py

```diff
@@ -902,6 +902,15 @@ def test_run_detector():
     assert len(responses.calls) == 2
 
 
+@responses.activate
+def test_run_detector_secondary_raster():
+    add_mock_detector_run_responses(1)
```

Comment:
Again, not from this PR, but I feel this function should also take and check the body (one or two raster ids)

Reply:
Good idea, done.